src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md

```diff
@@ -384,3 +354,19 @@ Note that on Windows, you also need to make sure that your drive in question, sa
 has been marked for sharing in the Docker settings:
 
 ![Running k6 in docker on Windows](images/Modules/running-k6-in-docker-on-windows.png)
+
+## The JSLib repository
+
+[JSLib](https://jslib.k6.io/) is a set of libraries known to play nicely with k6, and available at https://jslib.k6.io/.
+
+- `k6-utils`
+- `jsonpath`
+- `form-urlencoded`
+- `papaparse`
```

Comment:
```suggestion
```

I would not list these. It opens up a possibility for the docs to drift from the JSLib page, and as these increase, we'll have an increasingly difficult time of keeping them from diverging.